exercises/resistor-color-trio/description.md

```diff
@@ -9,16 +9,16 @@ If you want to build something using a Raspberry Pi, you'll probably use _resist
   In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take 3 colors as input, and outputs the correct value, in ohms.
   The color bands are encoded as follows:
 
-* Black: 0
-* Brown: 1
-* Red: 2
-* Orange: 3
-* Yellow: 4
-* Green: 5
-* Blue: 6
-* Violet: 7
-* Grey: 8
-* White: 9
+- Black: 0
```

Comment:
Why is this list using `-` whereas the same list in the above resistor-color-duo exercises uses `*`?

Reply:
Markdown lint rules say that all lists in a file should use only one or the other of * or - thought a file.
`markdownlint` prefers * when there are other issues in the list and no other lists already using -.
So, - would also be valid, but due to fixing other issues, markdown lint sometimes accepts - and sometimes requests *.
Technically either is acceptable as long as only one is used within each file.

Reply:
Okay, thanks for explaining!